Jenkinsfile

```diff
@@ -1,4 +1,25 @@
-elifeLibrary {
+import groovy.json.JsonSlurper
+
+@NonCPS
+def jsonToPypirc(String jsonText, String sectionName) {
+    def credentials = new JsonSlurper().parseText(jsonText)
+    echo "Username: ${credentials.username}"
+    return "[${sectionName}]\nusername=${credentials.username}\npassword=${credentials.password}"
+}
+
+def withPypiCredentials(String env, String sectionName, doSomething) {
+    try {
+        writeFile(file: '.pypirc', text: jsonToPypirc(sh(
+            script: "vault.sh kv get -format=json secret/containers/pypi/${env} | jq .data.data",
+            returnStdout: true
+        ).trim(), sectionName))
+        doSomething()
```

Comment:
the convention in Groovy for these closures is usually `it()`

Reply:
What does `it` stand for? Would suggest to me something like iterator or item (when iterating). Are you aware of a documentation of that convention?